client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx

```diff
@@ -0,0 +1,388 @@
+                                                        </Grid>
+                                                        <Grid item xs={5}>
+                                                            <FormControl>
+                                                                <RadioGroup
+                                                                    aria-label={OCCUPATION_LABEL}
+                                                                    name={OCCUPATION_LABEL}
+                                                                    value={interactedContact.occupation}>
+                                                                    {
+                                                                        occupations.map((occupation) => {
+                                                                            return <FormControlLabel
+                                                                                value={occupation}
+                                                                                key={occupation}
+                                                                                control={
+                                                                                    <Radio
+                                                                                        color='primary'
+                                                                                        onChange={(event) => {
+                                                                                            updateInteractedContact(interactedContact, InteractedContactFields.OCCUPATION, event.target.value)
+                                                                                        }}
+                                                                                    />
+                                                                                }
+                                                                                label={occupation}
+                                                                            />
+                                                                        })
+                                                                    }
+                                                                </RadioGroup>
+                                                            </FormControl>
+                                                        </Grid>
+                                                    </Grid>
+                                                </Grid>
+                                            </Grid>
+                                        </Grid>
+                                    </Grid>
+                                </AccordionDetails>
+                                <PrimaryButton
+                                    style={{ marginRight: '1.5vw' }}
```

Comment:
I know it's small but you should export it to your styles file

Reply:
doesn't work with className only style